setup.py

```diff
@@ -12,6 +12,7 @@
     packages=find_packages(),
     install_requires=[
         'torch',
+        'lm-eval',
```

Comment:
@HDCharles @cpuhrsch is this how we add dep for torchao?

Reply:
Let's make this an optional dependency for now. You only need that to train a quantizer for LLM's specifically. You don't have to add that by default.

Or put differently, I'd not add that righ tnow.

Reply:
we would break OSS CI without this, that's why I'm adding this here..

Reply:
Seems like that's just an import error. Yes, I'd disable that and throw a warning that it needs to be installed.

But on that note, the user shouldn't need `llm-eval` to run an already quantized model. So this is a good thing to fix in one of many follow ups :)